sdk/openai/Azure.AI.OpenAI/src/Custom/ChatMessage.cs

```diff
@@ -10,14 +10,18 @@ namespace Azure.AI.OpenAI
     /// <summary> A single, role-attributed message within a chat completion interaction. </summary>
     public partial class ChatMessage
     {
+        /// <summary> Initializes a new instance of ChatMessage. </summary>
+        public ChatMessage()
```

Comment:
Looking at the [API reference](https://platform.openai.com/docs/api-reference/chat/create), both "role" and "content" are required. I feel like this is new and it didn't use to work like this? I wonder if I'm missing something, but assuming the API reference is correct, I wonder if we should keep only the constructor that takes both parameters.